utils_nlp/models/transformers/common.py

```diff
@@ -39,6 +52,18 @@
 TOKENIZER_CLASS.update(
```

Comment:
I tried AutoTokenizer in the newest version of Transformers and found the code is much cleaner without specifically using such a dictionary.

Reply:
Oh I didn't know these exist - great. I'll try this and might also move to AutoModels while we're at it